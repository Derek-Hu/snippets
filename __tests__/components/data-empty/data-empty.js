import React from 'react';
import DataEmpty from '~/components/data-empty/index';

it('空白数据时组件', () => {
  const wrapper = mount(<DataEmpty />);

  expect(wrapper).toMatchSnapshot();
});
