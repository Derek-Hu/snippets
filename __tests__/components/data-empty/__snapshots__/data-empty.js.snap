// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`空白数据时组件 1`] = `
<_default>
  <OriginEmpty
    description="No Data Yet!"
    image="null.png"
    imageStyle={
      Object {
        "height": 40,
      }
    }
  >
    <LocaleReceiver
      componentName="Empty"
    >
      <div
        className="ant-empty"
      >
        <div
          className="ant-empty-image"
          style={
            Object {
              "height": 40,
            }
          }
        >
          <img
            alt="No Data Yet!"
            src="null.png"
          />
        </div>
        <p
          className="ant-empty-description"
        >
          No Data Yet!
        </p>
      </div>
    </LocaleReceiver>
  </OriginEmpty>
</_default>
`;
